vector/src/main/java/im/vector/app/features/spaces/create/SpaceDetailEpoxyController.kt

```diff
@@ -64,7 +64,6 @@ class SpaceDetailEpoxyController @Inject constructor(
             enabled(true)
             value(data?.name)
             hint(host.stringProvider.getString(R.string.create_room_name_hint))
-            singleLine(true)
```

Comment:
enter won't go to next field now ? why isn't it anymore single line

Reply:
`true` is now the default value